docs/basics/planning.md

```diff
@@ -2,127 +2,61 @@
-it easy to access the latest source-code, to submit pull requests
-for any patches you create and to collaborate with many other
-CiviCRM developers who also use GitHub.
-
-**Install the [buildkit](https://github.com/civicrm/civicrm-buildkit)**,
-ideally as a [vagrant virtual-machine](https://github.com/civicrm/civicrm-buildkit-vagrant)
-or using one of available `docker` images ([1](https://github.com/progressivetech/docker-civicrm-buildkit),
-[2](https://github.com/ErichBSchulz/dcbk)). The buildkit is not an absolute
-requirement but it is definitely the fastest path to a good development
-experience!
-
-**From the outset, [automate testing](/testing/setup.md)**.
-In the current climate of rapid evolution of not just CiviCRM, but
-also it's myriad of dependancies, automated testing of PHP code with `phpunit`
-and javascript with tools like `karma` and `jasmine` is essential. Start all
-your work by considering how you will provide automated testing for it.
-Starting with the buildkit will make this much simpler for you to set up.
-Getting started with unit-testing may seem daunting and onerous when you start,
-but you will soon come to love the freedom it gives you. If you are
-unsure how to proceed with testing ask the [community](/basics/community.md).
-
-**Create a native [extension](/extensions/index.md)**.
-If you have new functionality to add to CiviCRM, it probably belongs in an
-extension. "Native" extensions will install into all CiviCRM sites
-regardless of the  underlying CMS used (Drupal or Wordpress), making it easy to
-share your extension with the CiviCRM community.
-
-**Use the [API](/api/index.md) and [hooks](/hooks/index.md)** to access and manage CiviCRM data in any patch,
-native extension, CMS module, or external program that you
-develop. The API will function as expected with every new release
-and backwards compatibility of the API is maintained for several
-versions of CiviCRM.
-
-**Avoid [hacking the core](/core/hacking.md)** of CiviCRM unless you understand
-the implications.
-
-**Follow the
-[Coding Standards](/standards/index.md)**
-for uniform structure that will make everyone's development work easier.
+**Open a free [GitHub](https://github.com/) account** for revision control. The official CiviCRM [repositories](https://github.com/civicrm) are housed in `git` repositories on GitHub.  If you use GitHub you will find it easy to access the latest source-code, to submit pull requests for any patches you create and to collaborate with many other CiviCRM developers who also use GitHub.
+
+**Install the [buildkit](https://github.com/civicrm/civicrm-buildkit)**, ideally as a [vagrant virtual-machine](https://github.com/civicrm/civicrm-buildkit-vagrant) or using one of available `docker` images ([1](https://github.com/progressivetech/docker-civicrm-buildkit), [2](https://github.com/ErichBSchulz/dcbk)). The buildkit is not an absolute
+requirement but it is definitely the fastest path to a good development experience!
```

Comment:
@dvhirst should this be combined into the line above?

Reply:
Missed combining line 46 into 45.  Oops.

Reply:
Missed combining line 79 / 46 into 78 / 45.  Oops.